### ISSUE TITLE
Better toolbar

### ISSUE THREAD

**Issue by mattfidler** [ergoemacs/ergoemacs-mode#387]
Maybe variant of Notepad++ tool bar:

![image](https://cloud.githubusercontent.com/assets/514778/11901633/35fc8a80-a572-11e5-9cd7-b6e6b9a2a99a.png)

What do you think @xahlee

**Comment by mattfidler** [comment]
Other images:
![image](https://cloud.githubusercontent.com/assets/514778/11901746/ddaa7986-a572-11e5-8f62-0548366a1335.png)


![image](https://cloud.githubusercontent.com/assets/514778/11901779/07d34c24-a573-11e5-87d8-4325b14ea05e.png)

**Comment by mattfidler** [comment]
I think the icons are located at:

https://github.com/notepad-plus-plus/notepad-plus-plus/tree/master/PowerEditor/src/icons

**Comment by xahlee** [comment]
ugh, i think it's ugly... well, i think it's something i'd never consider, because what i had in mind of ideal emacs is, utmost efficiency.

adding a tool bar like that would be a different direction than my personal ideal. I think it would make emacs appear to some people... perhaps optional on/off, i don't see why not. On the other hand, the very existence of this bar, will become a laughing stock for many hard core emacses to point at ergoemacs, i think, even ergoemacs is far more hacker efficient than plain gnu emacs.

**Comment by mattfidler** [comment]
True.  Another similar project to ergoemacs-mode (as far as modernization) is aqua-emacs, which has also changed the tool bar.

I agree that ergoemacs-mode is also more hacker efficient, and hackers don't typically use a tool bar.  Both Sublime and Textmate didn't have a tool bar (as far as I can tell)

**Comment by mattfidler** [comment]
I also find macro recording from a button to be useful.

**Comment by mattfidler** [comment]
The other issue is the size of the icons.  The rest of emacs assumes the large sized images, so it may not work.
